src/format/KeePass2.cpp

```diff
@@ -21,13 +21,15 @@
 #include "crypto/kdf/Argon2Kdf.h"
 #include "crypto/CryptoHash.h"
 
-const Uuid KeePass2::CIPHER_AES = Uuid(QByteArray::fromHex("31c1f2e6bf714350be5805216afc5aff"));
-const Uuid KeePass2::CIPHER_TWOFISH = Uuid(QByteArray::fromHex("ad68f29f576f4bb9a36ad47af965346c"));
-const Uuid KeePass2::CIPHER_CHACHA20 = Uuid(QByteArray::fromHex("D6038A2B8B6F4CB5A524339A31DBB59A"));
+#define UUID_LENGHT 16
```

Comment:
Typo

Reply:
Yes, I realized that I missed to update some classes: I'm using Qt Creator and for some reason it didn't show them to me, also all the tests were ok.
Anyway I added the WIP tag because of that: there are still a lot of things to fix.
Thanks for signaling the typo: coding at night is not always a good idea :P